### DISCUSSION TITLE
Explore tab doesn't render GFM emoji

### DISCUSSION THREAD

**Discussion by TPS** [community/community#54092] [Mobile] [answered]
Choose the Explore tab in the app.  Anything recommended there that shows a GitHub-flavor Markdown emoji doesn't render as such, but shows `:whatever:`.  When choosing   the link to the item's page, it renders correctly there.

![042d6ee9-a0b2-4586-a71c-f6d75e856b7f.jpg](https://user-images.githubusercontent.com/185902/235358542-659bea86-ba89-4949-8dbc-1b4b5acce173.jpg)

**Comment by mattruggio** [comment]
Hey @TPS, thanks for the heads up!  Unfortunately I am having a hard time reproducing this on my mobile device and do see GFM-style emojis in my explore feed 😞. Could you provide us with the platform/version of the GitHub mobile app you are using (for example: GitHub for iOS, Version A.BCD.E (123456789))?

**Reply by TPS** [reply]
I'm on GitHub Mobile app for Android, v1.112.0

> Unfortunately I am having a hard time reproducing this on my mobile device and do see GFM-style emojis in my explore feed 😞.

Are you certain you're seeing GFM or regular Unicode emoji?  It's relatively rare to see the GFM kind.

**Reply by mattruggio** [reply]
Ah, you are correct!  I am going to get this internally filed as a bug.

**Comment by elisealix22** [comment, marked as answer]
@TPS we just released a new version that supports emoji shortcodes throughout our Android app. Thanks for putting this on our radar!

**Reply by TPS** [reply]
@elisealix22 I'll take your word for it.  If it works, I'll never see unrendered GFM emoji in Explore again, but I'll report back if I ever do.
